src/plccng/scanner/run.py

```diff
@@ -0,0 +1,15 @@
+from .main import Main
+from .scanner import Scanner
+from .source import Source
+from .matcher import Matcher
+from .structs import HasLexErrors
+
+def run(args):
+	try:
+		Main(Scanner, Source, Matcher).run(args)
+	except HasLexErrors as errors:
```

Comment:
LexErrors need to be inline. That is, we should scan and and print tokens; if we encounter a lex error, we print it and continue scanning for more tokens.